### ISSUE TITLE
Forbidden error

### ISSUE THREAD

**Issue by fisherds** [GoogleCloudPlatform/appengine-endpoints-tictactoe-python#7]
There was a change to make line 46 of the tictactoe.html file from:

```
      google.devrel.samples.ttt.init('https://' + window.location.host + '/_ah/api');
```

to...

```
      google.devrel.samples.ttt.init('//' + window.location.host + '/_ah/api');
```

However without the https Chrome defaulted to http instead of https which caused the tictactoe client to not load.  Not sure if others have the same error, but that change broke my web client.  I added back the https in my local version and life was happy again.  Just sharing in case others have the issue too.  I'm sure it fixed some people's issues, but this change may have broken more people. :)

**Comment by dhermes** [comment]
Thanks for this @fisherds. The intention is to also have `secure: always` in each handler directive, but that was an oversight by us. By doing this `http` is not a possible protocol for users to interact with.

**Comment by dhermes** [comment]
Fixed in 263fb21c251efd10c6ca193848c1c35e161cde7f
